web/src/components/CardHeader.vue

```diff
@@ -3,20 +3,21 @@
     <h2>{{ title }}</h2>
     <v-spacer />
     <v-btn
+      v-if="false"
```

Comment:
I caught this mistake though it's a bit outside the PR. The button was rendering even though the icon is disabled.